### DISCUSSION TITLE
Customized model creation accessible to users

### DISCUSSION THREAD

**Discussion by Genai-labs** [open-webui/open-webui#4589] [Ideas]
#### Your feature request related to a problem? Please describe.
It’s not a problem, but rather related to the idea of making this app available to a group of people.

#### Describe the solution you’d like
In the case of a large number of users, it could be interesting if they could create their own customized models without having to go through the admins.

**Comment by elharony** [comment]
I completely support this feature request. Currently, creating custom models is limited to admins, which isn't ideal. It would be better if regular users could create their own models without admin rights.

Moreover, allowing users to create "me-only" custom models is also important to avoid sharing all models with everyone, which isn't always appropriate. Additionally, it would be useful if users could create models for broader use, but with admin approval required before they become accessible to others.

**Comment by PatBQc** [comment]
Hello,

Just reaching out to see if something moved on this topic or if someone found a workaround solution ?

We deployed OpenWebUI to a 100+ users and a request often asked from the users is to have the ability to create GPTs (in their own words) that would be Custom Models in OpenWebUI terminology.

We want to keep the admin role as closed as possible and would love if it was possible to determine per user who (role based I guess) can create new custom models and who can share them.

Otherwise, WE LOVE OPENWEBUI !!!  Kudos to all the team working on this !

**Comment by PatBQc** [comment]
BTW, I created an issue yesterday and it was marked as resolved in v0.4 !  🎉🎉🎉
